src/views/ProfileView.vue

```diff
@@ -111,6 +114,9 @@ export default {
     closeAnnouncementModal() {
       this.openModal = false;
     },
+    deleteAnnouncement() {
+      this.announcementsListKey += 1; // reloads the announcements-list component
+    },
```

Comment:
This is a clever solution for the component as it is. Needing to do this though is a hint that we should refactor the underlying methodology of grabbing announcements out into the store.